outreach/gsoc/2024/idea-list.md

```diff
@@ -23,11 +23,16 @@ Currently Wayshot performs image capturing and transformations through the CPU,
 
 This idea will implement a DMA-BUF backend and a client-side API for high performance image capturing and transformations on the GPU. The new backend and api should enable high performance streaming over applications like [OBS](https://github.com/obsproject/obs-studio) using [our custom desktop portal backend](https://github.com/waycrate/xdg-desktop-portal-luminous).
 
+### Expected Outcomes
+
+- Able to write a DMA-BUF backend in Wayshot
```

Comment:
I'd like these two points to be exchanged with the following:

- Integration of DMA-BUF backend into wayshot will allow high performance screen capture and possibly extend [xdg-desktop-portal-luminous](https://github.com/waycrate/xdg-desktop-portal-luminous)

Reply:
Done